spec/js/no_in_person_locations_display_spec.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @jest-environment jsdom
+ */
+
+describe('NoInPersonLocationsDisplay', () => {
+  beforeEach(() => {
+    document.body.innerHTML = `
+      <no-in-person-locations-display
+        address="Somewhere over the rainbow"
+      ></no-in-person-locations-display>
+    `;
+  });
+
+  it('renders h2 status text', () => {
+    const noInPersonLocationsDisplay = document.querySelector('no-in-person-locations-display')!;
+
+    expect(noInPersonLocationsDisplay.hasAttribute('status'));
+    expect(noInPersonLocationsDisplay.hasAttribute('h2'));
```

Comment:
So, I was still confused by how `<no-in-person-locations-display>` was expected to do anything, since that would imply a [custom element](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements) (which isn't what we've implemented) and otherwise there's nothing to initialize the React.

I think the problem is that these tests aren't actually asserting anything, so they're passing, and would pass with whatever attribute parameter you might put here.

What it should be is:

```suggestion
    expect(noInPersonLocationsDisplay.hasAttribute('status')).toBe(true);
    expect(noInPersonLocationsDisplay.hasAttribute('h2')).toBe(true);
```

Reply:
I was not able to get .toBe(true) working. I opened up [PR 1171](https://github.com/18F/identity-site/pull/1171) as a result of this comment.